File: Microsoft.Recognizers.Text.NumberWithUnit/Spanish/Parsers/VolumeParserConfiguration.cs
using Microsoft.Recognizers.Text.NumberWithUnit.Spanish.Extractors;
using Microsoft.Recognizers.Text.Number.Utilities;
using System.Globalization;

namespace Microsoft.Recognizers.Text.NumberWithUnit.Spanish.Parsers
{
    public class VolumeParserConfiguration : SpanishNumberWithUnitParserConfiguration
    {
        public VolumeParserConfiguration() : this(new CultureInfo(Culture.Spanish)) { }

        public VolumeParserConfiguration(CultureInfo ci) : base(ci)
        {
            this.BindDictionary(VolumeExtractorConfiguration.VolumeSuffixList);
        }
    }
}

File: Java/libraries/recognizers-text-number-with-unit/src/main/java/com/microsoft/recognizers/text/numberwithunit/portuguese/parsers/VolumeParserConfiguration.java
package com.microsoft.recognizers.text.numberwithunit.portuguese.parsers;

import com.microsoft.recognizers.text.Culture;
import com.microsoft.recognizers.text.CultureInfo;
import com.microsoft.recognizers.text.numberwithunit.portuguese.extractors.VolumeExtractorConfiguration;

public class VolumeParserConfiguration extends PortugueseNumberWithUnitParserConfiguration {

    public VolumeParserConfiguration() {
        this(new CultureInfo(Culture.Portuguese));
    }

    public VolumeParserConfiguration(CultureInfo cultureInfo) {
        super(cultureInfo);

        this.bindDictionary(VolumeExtractorConfiguration.VolumeSuffixList);
    }
}
